core/domain/param_domain_test.py

```diff
@@ -106,7 +102,7 @@ def test_param_change_class(self):
             'generator_id': 'Copier',
             'customization_args': {'value': '3'}
         })
-        self.assertEqual(param_change.get_normalized_value('Int', {}), 3)
+        self.assertEqual(param_change._get_value({}), '3')   # pylint: disable=protected-access
```

Comment:
We usually try to limit the number of pylint disables, can yo utry to remove the need for it here? Or explain why it is absolutely needed.

Reply:
We need the pylint disable for the method name `_get_value` as it starts with `_`. I think we have two options, either rename the method name or keep the pylint disable. what you would suggest

Reply:
I think better rename it.

Reply:
done